### ISSUE TITLE
[Bug] Latest playground is broken on Keychron Q6 (at least). Keys messed up/swapped.

### ISSUE THREAD

**Issue by richard-browne** [Keychron/qmk_firmware#259]
### Describe the Bug

I built the latest on playground with `make keychron/q6/ansi_encoder:keychron`. After installing the firmware many keys were messed up. Control and fn are swapped, all keys on the asdf row are off by one (e.g. a key gives 's', s key gives 'd' etc).

I went back to an earlier commit and that worked fine.

### Keyboard Used

Q6

### Operating System

n/a

### Is AutoHotKey / Karabiner installed

- [ ] AutoHotKey (Windows)
- [ ] Karabiner (macOS)

**Comment by lokher** [comment]
Is your Q6 with knob or without knob? The knob version use `make keychron/q6/ansi_encoder:keychron`, or else use `make keychron/q6/ansi:keychron`

**Comment by richard-browne** [comment]
Yes I have the knob version. I built the latest on old_playground and that worked fine.

I wish Keychron would tag the commits corresponding to the official firmware releases.

**Comment by PeterMortensen** [comment]
"playground" is ambiguous. There are these branches:

 * *[wireless_playground](https://github.com/Keychron/qmk_firmware/tree/wireless_playground/keyboards/keychron)* (probably the most current. [Last commit/merge](https://github.com/Keychron/qmk_firmware/commits/wireless_playground/) [on 2024-05-21](https://github.com/Keychron/qmk_firmware/commit/b6d42d84d2726e3976508dac48f937b2502f03e8).)

 * *[bluetooth_playground](https://github.com/Keychron/qmk_firmware/tree/bluetooth_playground/keyboards/keychron)*  (probably more or less stale, though it has [received more testing](https://github.com/Keychron/qmk_firmware/issues/217#issuecomment-1982186367) than 'wireless_playground') 

 * *[playground](https://github.com/Keychron/qmk_firmware/tree/playground/keyboards/keychron)*. Last commit 2024-01-27 (?).

 * *[old_playground](https://github.com/Keychron/qmk_firmware/tree/old_playground/keyboards/keychron)*. Last commit 2023-04-22 (?).

Which one? The literal "[playground](https://github.com/Keychron/qmk_firmware/tree/playground/keyboards/keychron)"? It seems to be stale.

Though there is never an official unambiguous statement of any kind about (the state of) the branches.

Related: *[The current software state](https://github.com/Keychron/qmk_firmware/issues/217)* (2024-01-16)

**Comment by richard-browne** [comment]
Keychron should do better. There's no way to know what branch/commit to build for your keyboard. It's a mess.

**Comment by PeterMortensen** [comment]
Is there a reason not to use [the official QMK repository](https://github.com/qmk/qmk_firmware/tree/master/keyboards/keychron/q6)? The [raw HID](https://www.pjrc.com/teensy/rawhid.html) commands? Support by [Keychron's Via clone](https://www.reddit.com/r/Keychron/comments/1buwibx/comment/kxx3kt0/)? Something else?

It was my impression the (wired-only) Q series was kept most up-to-date in the official QMK repository.
